src/components/RenderRouter/FormItem.tsx

```diff
@@ -17,11 +18,24 @@ export default class ContentItem extends React.Component<Props, State>  {
     }
   }
 
+  ogUrl(){
```

Comment:
If you use: this.imgUrl(480)+image1.src it will automatically resize whatever image1.src is...imgUrl(horizontalSize)..this only works on the server, but that way we don't need to duplicate images..